indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -168,6 +172,19 @@ public ForkingTaskRunner(
     return retVal;
   }
 
+  public void registerListener(TaskRunnerListener listener, Executor executor)
```

Comment:
I didn't have a need for that, IMO no point adding a method until someone needs it